lib/auth/v2/headerAuthCheck.js

```diff
@@ -21,10 +21,9 @@ function check(request, log, data) {
           'x-amz-date header') };
     }
 
-    const timeout = checkRequestExpiry(timestamp, log);
-    if (timeout) {
-        log.debug('request time too skewed', { timestamp });
-        return { err: errors.RequestTimeTooSkewed };
+    const timeoutResult = checkRequestExpiry(timestamp, log);
```

Comment:
following my previous comment, this could be changed into:
```
const err = checkRequestExpiry(timestamp, log);
if (err) {
    return { err } ; // Uses the shorthand syntax for the wrapping (not necessary)
}
```